apps/functions-subscription/src/adapters/azure/cosmosdb/__tests__/subscription.test.ts

```diff
@@ -0,0 +1,82 @@
+import { describe, expect, it } from 'vitest';
+import * as E from 'fp-ts/lib/Either';
+import * as O from 'fp-ts/lib/Option';
+import { Database } from '@azure/cosmos';
+import { ErrorResponse } from '@azure/cosmos';
```

Comment:
**suggestion**: you can join this imports

**question**: where does the `@azure/cosmos` dependency come from? I don't see it listed in the `package.json`